### ISSUE TITLE
First post

### ISSUE THREAD

**Issue by ineiti** [dedis/paper_chainiac#1]
I added an `Rx` and `Tx` dummy function to
`/api.go`
`/swupdate/service/api.go`
`/timestamp/client_api.go`
And wonder: should this be included in `onet.Client`? Others might want to use it, too, and it wouldn't break anything existing...

**Comment by nikkolasg** [comment]
Yes, there is already an issue for that https://github.com/dedis/onet/issues/96
`onet.Client` is the right place for that I believe too !

**Comment by ineiti** [comment]
+1 - thanks. Do you have time to do that today? I think for the chainiac-measurements this would be needed. So either one of us should do this.
